src/components/VersionSelect/index.tsx

```diff
@@ -1,15 +1,20 @@
 import * as React from 'react';
 import { Col, Form } from 'react-bootstrap';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Why is this needed?

Reply:
`FormControlProps` is required by TS and only available from `react-bootstrap/lib/FormControl`, but ESLint cannot find it that way (it does not resolve the `import` correctly apparently). I could not find a better solution and did not find any useful issue about that on the ESLint plugin repo :(